registry/handlers/app.go

```diff
@@ -640,6 +640,7 @@ func (app *App) dispatcher(dispatch dispatchFunc) http.Handler {
 		}
 
 		context := app.context(w, r)
+		defer defaultContextManager.release(context)
```

Comment:
Hmm... good question. We don't track number of references, so this might have unexpected consequences...